rebar.config

```diff
@@ -20,3 +20,13 @@
 {profiles, [{test,
              [{erl_opts, [debug_info]}
             ]}]}.
+
+{relx, [{release, { aesophia_cli, "7.0.1" },
+              [sasl, aesophia_cli, aesophia, aeserialization, eblake2]},
+
+        {dev_mode, false},
+        {include_erts, true}, % To include Erlang runtime system
```

Comment:
If it is possible to include Erlang runtime into executable, why do we need warp in macOS and Linux?

Reply:
It's not a single executable like a C program. It creates a self extracting tar file with the command